onnx2torch/node_converters/slice.py

```diff
@@ -119,7 +126,19 @@ def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint:
 @add_converter(operation_type='Slice', version=11)
 @add_converter(operation_type='Slice', version=13)
 def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint: disable=unused-argument
+    constant_params = []
+    params_names = node.input_values[1:]
+    for name in params_names:
+        try:
+            constant_params.append(get_const_value(name, graph).tolist())
+        except KeyError:
+            constant_params.append([])
+    constant_params = constant_params + (4 - len(constant_params)) * [[]]
```

Comment:
why is this needed?

Reply:
`onnx.slice` takes in 5 inputs: `data`, `starts`, `ends`, `axes` (optional), `steps` (optional).
For example, if the value of `steps` is not given as the input to a `onnx.slice` node, then the length of `node.input_values[1:]` will just be 3, but `constant_params` needs to have four elements to instantiate the converted torch module.